Configs/.local/lib/hyde/wl-togglesink.sh

```
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+scrDir="$(dirname "$(realpath "${0}")")"
+source "${scrDir}/globalcontrol.sh"
+dunstDir="${iconsDir}/Wallbash-Icon"
```

Shellcheck (0.11.0)

[warning] iconsDir is referenced but not assigned.

(SC2154)